### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by rachelsoae**
Hi all! Thanks for sending the bullet points for the README. I tried to incorporate everyone's thoughts into the final draft. Please comment with any changes you'd like to request and I'll get them done ASAP. I added all three of you as reviewers so we can make sure it's perfect.
Thanks!!

### CODE REVIEW

**Review by JamieCaudill — APPROVED** [review]
Looks great! 
The only thing I noticed is that you left the prompts in. I think I remember deleting those in the past projects. If you decide to remove them or not, I'm sure it'll be great!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+3. `cd` into the directory
+4. Run `open index.html` to launch the application in your default browser
+
+### Preview of App:
+[//]: <> (Provide ONE gif or screenshot of your application - choose the "coolest" piece of functionality to show off.)
+![scroll functionality](https://imgur.com/MikTsSn.gif)
+
+### Context:
+[//]: <> (Give some context for the project here. How long did you have to work on it? How far into the Turing program are you?)
+This project was assigned in Week 4 of Module 1 and we were given 8 days to complete it. As a group, we spent a total of 12 hours together working on the styling and functionality of the required features. We each spent several additional hours working independently to do research, generate ideas, and pseudocode solutions before bringing them to schedyked group meetings and were able to completet the base project requirements in just 5 days.
```
Comment (fernandorobles97) [inline README.md:20]: Looks like there are two typos here. "schedyked" and "completet"

**Review by fernandorobles97 — COMMENTED** [review]
This is an amazing summary of the overall project! There are two typos we can fix but besides that it is perfect.

**Review by vsriqui — APPROVED** [review]
A well written readme.

**Pull request merged**